hack/make/binary

```diff
@@ -36,7 +36,7 @@ if [ "$(go env GOOS)" == "linux" ] ; then
 	esac
 fi
 
-if [ "$IAMSTATIC" == "true" ] && [ "$(go env GOHOSTOS)" == "linux" ] && [ "$DOCKER_EXPERIMENTAL" ]; then
+if [ "$IAMSTATIC" == "true" ] && [ "$(go env GOHOSTOS)" == "linux" ]; then
```

Comment:
Edit: I tried this and it seems like it just complicates readability because of the complexity between `$IAMSTATIC` and `GOHOSTOS` vs. `GOOS`.  I can refactor if you'd prefer into the switch statement but I actually think the current separation might be preferred.